components/icon/web-component/stories/default.stories.mdx

```diff
@@ -39,10 +39,6 @@ import { IconExample } from "../template";
       icon: "utrecht-icon-loupe",
       size: 32,
     }}
-    parameters={{
-      // This is an interactive example where styling is unimportant, ignore this test
-      percy: false,
```

Comment:
I'm not sure if this should now be `chromatic: { disableSnapshot: true },` or not

Reply:
@Robbert was this a different way of disabling percy tests for this component?